specification/web/resource-manager/Microsoft.Web/2016-08-01/WebApps.json

```diff
@@ -13221,6 +16065,111 @@
         }
       }
     },
+    "FunctionEnvelope": {
+      "description": "Web Job Information.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/ProxyOnlyResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "description": "FunctionEnvelope resource specific properties",
+          "properties": {
+            "name": {
+              "description": "Function name.",
+              "type": "string"
+            },
+            "functionAppId": {
+              "description": "Function App ID.",
+              "type": "string"
+            },
+            "scriptRootPathHref": {
+              "description": "Script root path URI.",
+              "type": "string"
+            },
+            "scriptHref": {
+              "description": "Script URI.",
+              "type": "string"
+            },
+            "configHref": {
+              "description": "Config URI.",
+              "type": "string"
+            },
+            "secretsFileHref": {
+              "description": "Secrets file URI.",
+              "type": "string"
+            },
+            "href": {
+              "description": "Function URI.",
+              "type": "string"
+            },
+            "config": {
+              "$ref": "#/definitions/Object",
+              "description": "Config information."
+            },
+            "files": {
+              "description": "File list.",
+              "type": "object",
+              "additionalProperties": {
+                "type": "string"
+              }
+            },
+            "testData": {
+              "description": "Test data.",
+              "type": "string"
+            }
+          },
+          "x-ms-client-flatten": true
+        }
+      }
+    },
+    "FunctionEnvelopeCollection": {
+      "description": "Collection of Kudu function information elements.",
+      "required": [
+        "value"
+      ],
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "Collection of resources.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/FunctionEnvelope"
+          }
+        },
+        "nextLink": {
+          "description": "Link to next page of resources.",
+          "type": "string"
+        }
+      }
+    },
+    "FunctionSecrets": {
```

Comment:
Yes.  See [Functions-API](https://github.com/projectkudu/kudu/wiki/Functions-API) from the Kudu wiki